lib/generators/curation_concerns/templates/catalog_controller.rb

```diff
@@ -1,7 +1,7 @@
 class CatalogController < ApplicationController
   include CurationConcerns::CatalogController
   configure_blacklight do |config|
-    config.search_builder_class = CurationConcerns::SearchBuilder
+    # config.search_builder_class = ::SearchBuilder
```

Comment:
Yes, that mirrors what is provided in the default Hydra CatalogController template